### PR TITLE
﷽

### PR DESCRIPTION
**Pull request by RuyiLi**
﷽﷽﷽

### CODE REVIEW

**Review by ConnorMcF — APPROVED** [review]
Literally unplayable, this fix resolves all issues and fixes bot uptime.

**Review by matmen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ Invite/OAUTH: `https://discordapp.com/oauth2/authorize?client_id=170903265565736
 I won't mind if you take some code to learn or improve but please don't be a skid.
 Feel free to ask me questions, NotSoSuper#8800 on Discord.
 
-The MySQL database scheme is included in `discord.sql`, you will have to reverse or replace the enviormental variables in the code for proper results.
+The MySQL database scheme is included in `discord.sql`, you will have to reverse or replace the environmental variables in the code for proper results.
```
Comment (ConnorMcF) [inline README.md:11]: whom say environmental variables, environment variables please and thanks

**Review by ConnorMcF — CHANGES_REQUESTED** [review]

**Comment by Dominic18** [comment]
You can go fuck yourself.

**Comment by RuyiLi** [comment]
huh? 💀

**Pull request closed without merging**